### PR TITLE
Pindel

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ea9b0d395ae2958cad04 for a full test report.

File: easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'CmdCp'
+
+name = 'Pindel'
+version = '0.2.5a7'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb]: if interested, `0.2.5` now exists

File: easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+dependencies = [('SAMtools', '0.1.19')]
+
+cmds_map = [('.*', "CPATH=$EBROOTSAMTOOLS/include/bam:$CPATH ./INSTALL $EBROOTSAMTOOLS")]
+ 
+files_to_copy = [(['pindel', 'sam2pindel', 'pindel2vcf', 'bam2pindel.pl', 'Adaptor.pm'], 'bin'),
+                "demo", "gmt", "gmt-web", "test", "README.md", "RELEASE"
+                ]
+
+postinstallcmds = ["sed -i -e 's|/usr/local/bin/perl|/usr/bin/env\ perl|' %(installdir)s/bin/bam2pindel.pl"]
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb]: ping on this bit?

File: easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb (hunk 1 above)
Comment (pescobar) [inline easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb]: 0.2.5a7 is the latest release https://github.com/genome/pindel/releases

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b384ce9b9cc4460d8b4a for a full test report.

File: easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb (hunk 1 above)
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb]: (!)

I stand corrected!

On 3 June 2015 at 20:04, Pablo Escobar Lopez <notifications@github.com>
wrote:

> 0.2.5a7 is the latest release https://github.com/genome/pindel/releases
>


-- 


echo "sysadmin know better bash than english"|sed s/min/mins/ \
  | sed 's/better bash/bash better/' # signal detected in a CERN forum

**Comment by boegel** [comment]
@pescobar: Jenkins detected a conflict:

```
Conflict found for dependencies of Pindel-0.2.5a7-goolf-1.4.10: zlib-1.2.7-goolf-1.4.10 vs zlib-1.2.8-goolf-1.4.10
```

File: easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/genome/pindel/archive/']
+sources = ['v%(version)s.tar.gz']
+
+patches = ['pindel-perl-path.patch']
+
+dependencies = [
+    ('SAMtools', '0.1.19'),
+    ('zlib', '1.2.8')
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pindel/Pindel-0.2.5a7-goolf-1.4.10.eb]: SAMtools already depends on zlib 1.2.7, so use 1.2.7 here too

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/48247d7f885a7c627524 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**